.circleci/config.yml

```diff
@@ -96,6 +98,13 @@ references:
         # Max 4 threads to not overtax CI.
         python${PYVER} setup.py build_ext --inplace -- -DCOVERAGE=ON -- -j 4 -v
 
+  build_windows: &build_windows
+    run:
+      name: Build with MSVC
+      command: |
+        python${PYVER} --version
+        python${PYVER} setup.py build_ext --inplace -- -G "Visual Studio 16 2019" -DCOVERAGE=ON
```

Comment:
I think we could simplify this later, since `build_windows` and `build` are now the same except for some platform-specific flags. I would prefer to cut the scope where it is, and do that in #668 or afterward.

Reply:
I'd say that's a post-skbuild refactor, I've noticed in a couple recent iterations that our CI scripts in general could use some cleanup. We can do it when we also do things like enabling pre-commit and clang-format.